org-formation/800-redirects/_tasks.yaml

```diff
@@ -35,3 +37,22 @@ FinopsApiRedirect:  # redirect finops-api.sageit.org to lambda-mips-api cloudfro
     SourceHostedZoneId: "Z0478495257GEB73WFM63"
     # the value of the CNAME record
     TargetHostName: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-mips-microservice-CloudfrontDomain', !Ref AdminCentralAccount]
+
+
+# *.app.sagebionetworks.org
+
+DcaDevAppRedirect:  # redirect dca-test.app.sagebionetworks.org to data_curator-infra ALB
```

Comment:
@jesusaurus can you confirm that, though you use the term 'redirect' there isn't literally an HTTP redirect (3xx response), but rather traffic is forwarded to the target ALB?

Reply:
@brucehoff correct, this is not an HTTP redirect, the redirection is happening when looking up the IP address of the host name (which I have an old habit of also calling a redirect, but I can change that terminology if its confusing)

Reply:
could we add a comment referencing the GH repo for this redirect?

Reply:
@zaro0508 done